react/src/pages/MainMenu/MainMenu.tsx

```diff
@@ -32,12 +33,20 @@ function MainMenu() {
     <>
       <SectionHeader isNestedHeader>Main Menu</SectionHeader>
       <InlineMessage type="success">
-        Welcome, {userData?.username || 'User'}
+        Welcome, {userData?.username || 'User'} <Icon name="user"></Icon>
       </InlineMessage>
 
       <table>
-        <thead>Projects</thead>
-        <tbody>You have {data?.length} projects.</tbody>
+        <thead>
+          <tr>
+            <th>Projects</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>You have {data?.length} projects.</td>
+          </tr>
+        </tbody>
       </table>
```

Comment:
**Yes, great!** Accurate markup for tables is important to apply Core-Styles to them.